src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ cloud:
   aws:
     region:
       static: ${AWS_REGION}
+    cloudwatch:
+      namespace: TIS/Trainee/UserManagement
+      awsAccessKey: ${AWS_ACCESS_KEY_ID:}
```

Comment:
I am sure there is a better way to configure this service so that these are not required 🤷

Reply:
🤔 Does it not use the usual credentials provider chain? That usually pulls from the env var without assigning it to a param

Reply:
At the moment it needs to pull them in the cloudWatchAsyncClient config:
```
StaticCredentialsProvider.create(new AwsCredentials() {
              @Override
              public String accessKeyId() {
                return awsAccessKey;
              }

              @Override
              public String secretAccessKey() {
                return awsSecret;
              }
```

Reply:
I'll have a tinker in the morning

Reply:
Ok, I think I've resolved this.